src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs

```diff
@@ -239,25 +239,24 @@ private static int GetWindowsBuildNumber()
         );
 
         [DllImport("ntdll.dll", ExactSpelling=true)]
-        private static extern int RtlGetVersion(out RTL_OSVERSIONINFOEX lpVersionInformation);
+        private static extern int RtlGetVersion(ref RTL_OSVERSIONINFOEX lpVersionInformation);
 
-        [StructLayout(LayoutKind.Sequential)]
-        private struct RTL_OSVERSIONINFOEX
+        [StructLayout(LayoutKind.Sequential, CharSet=CharSet.Unicode)]
+        private unsafe struct RTL_OSVERSIONINFOEX
```

Comment:
Why don't we include `src/Common/src/Interop/Windows/NtDll/Interop.RTL_OSVERSIONINFOEX.cs` to avoid the duplication here?

Reply:
This file has duplicate local interop definitions for everything it calls - CloseHandle, GetCurrentProcess, ... . I do not know why it was done this way. If there is a good reason to clean it up, it should be separate PR.

Reply:
https://github.com/dotnet/corefx/issues/32438